app/components/ui/tale-tab-metadata/component.js

```diff
@@ -27,19 +28,37 @@ export default Component.extend({
       component.set('environments', images);
       component.selectDefaultImageId();
     });
+
+    // Fetch licenses for users to select
+    $.getJSON(this.get('apiHost') + '/api/v1/license/').then(function(licenses) {
+      component.set('licenses', licenses);
+      component.selectDefaultLicense();
+    });
```

Comment:
Missing error-handling here, but probably not necessary in this case (unless `GET /license` is a more complex endpoint than it seems)

Reply:
@ThomasThelen please address this question.  Did you choose jQuery for a reason?

Reply:
I think I chose it based on the call to the images a few lines higher in the source; I'll make a query to the store though to keep things as Ember as possible